test/utils/utils.hpp

```diff
@@ -27,3 +28,39 @@ inline bytes operator""_hex(const char* s, size_t size)
 {
     return from_spaced_hex({s, size}).value();
 }
+
+/// Translates tests fork name to EVM revision
+inline evmc_revision to_rev(std::string_view s)
```

Comment:
i think this still should be inside `evmone::test` namespace (although other things in this files are not).
Moreover, we should move the implementation to `utils.cpp` because we often need to drop new entry here for testing. This will save us rebuilding most of the files.
Finally, it would be nice to split this change into separate commit. I can do this in the end.

Reply:
Hi @chfast, thanks for all your comments and explanations, they were very helpful. I tried to implement the separation you mention in this comment. However, I always get an 'undefined reference to `evmone::test::to_rev(...)', so clearly I'm missing something and I don't see what it could be.

Sorry, my knowledge of C++ is not something that stands out haha.

Reply:
You're not compiling `utils.cpp` currently, add it to `test/utils/CMakeLists.txt`
(also you'll probably need to add library dependency `eoftest` - > `testutils`)

Reply:
> You're not compiling `utils.cpp` currently, add it to `test/utils/CMakeLists.txt`
> (also you'll probably need to add library dependency `eoftest` - > `testutils`)

Oh, right, the CMakeList! I always forget them 😞. Thanks!